src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemPartition.tsx

```diff
@@ -27,19 +28,25 @@ const FileSystemPartition = () => {
         }
         name="sc-radio-automatic"
         description="Automatically partition your image to what is best, depending on the target environment(s)"
-        isChecked={fileSystemPartition === 'automatic'}
+        isChecked={fileSystemPartitionMode === 'automatic'}
         onChange={() => {
           dispatch(changeFileSystemPartitionMode('automatic'));
+          dispatch(changeFileSystemConfiguration([]));
         }}
       />
       <Radio
         id="manual file system config radio"
         label="Manually configure partitions"
         name="fsc-radio-manual"
         description="Manually configure the file system of your image by adding, removing, and editing partitions"
-        isChecked={fileSystemPartition === 'manual'}
+        isChecked={fileSystemPartitionMode === 'manual'}
         onChange={() => {
           dispatch(changeFileSystemPartitionMode('manual'));
+          dispatch(
+            changeFileSystemConfiguration([
+              { id: '1', mountpoint: '/', min_size: '500' },
```

Comment:
Looking good! Just one nit pick - can we use a generated uuid here so it's consistent with all the other partitions?